src/default-listeners.coffee

```diff
@@ -80,16 +79,34 @@ module.exports = (robot, config) ->
       # Disregard ignored usernames.
       return if msg.message.user.name in config.ignoreList
 
+      # Disregard any messages that have keywords
+      for phrase in config.ignoreMessageList
+        return if msg.message.text.indexOf(phrase) isnt -1
+
       # Pass the message to each active model.
       for name in activeModelNames
         robot.markov.modelNamed name, (model) -> model.learn msg.message.text
 
+    if config.learningListenMode == 'hear-all'
+      robot.hear //i, learning_listener
```

Comment:
This line is failing the tests. It looks like it doesn't parse:

```
[Fri Jun 05 2020 21:26:02 GMT+0000 (Coordinated Universal Time)] ERROR Unable to load /home/runner/work/hubot-markov/hubot-markov/src/index: /home/runner/work/hubot-markov/hubot-markov/src/default-listeners.coffee:91:21: error: unexpected ,
      robot.hear //i, learning_listener
```

Would this work if it was `/.*/i` instead?